### PR TITLE
fix: update yml and .env files

### PR DESCRIPTION
**Pull request by saalim-mushtaq**
### **PR Type**
Enhancement, Configuration changes

___

### **Description**
- Added new domain environment variables for admin, customer, and reseller portals in `.env`.
- Added a new GitHub Actions workflow for PR-Agent in `.github/workflows/pr_agent_codium.yml`.
- Updated PostgreSQL image version and added services for admin, customer, and reseller portals in `docker-compose.yml`.
- Added Google Analytics key environment variable in `web-api.env`.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>.env</strong><dd><code>Add new domain environment variables</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.env

<li>Added new domain environment variables for admin, customer, and <br>reseller portals.<br>

</details>

  </td>
  <td><a href="https://github.com/cryptlex/cryptlex-on-premise/pull/10/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055d">+3/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>web-api.env</strong><dd><code>Add Google Analytics key environment variable</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

web-api.env

- Added Google Analytics key environment variable.

</details>

  </td>
  <td><a href="https://github.com/cryptlex/cryptlex-on-premise/pull/10/files#diff-7b55bbe0c904bce7eb50e59bf9b79939275150685de8ee1d683fd7524b65ac89">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>pr_agent_codium.yml</strong><dd><code>Add PR-Agent GitHub Actions workflow</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.github/workflows/pr_agent_codium.yml

- Added a new GitHub Actions workflow for PR-Agent.

</details>

  </td>
  <td><a href="https://github.com/cryptlex/cryptlex-on-premise/pull/10/files#diff-31f7a07829ab13c46cb64fc7fa71ce36b42972e46c6de1722810696df0c2348a">+9/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>docker-compose.yml</strong><dd><code>Update PostgreSQL version and add new services</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docker-compose.yml

<li>Updated PostgreSQL image version.<br> <li> Added services for admin, customer, and reseller portals.<br>

</details>

  </td>
  <td><a href="https://github.com/cryptlex/cryptlex-on-premise/pull/10/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3">+75/-2</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -150,6 +150,79 @@ services:
       - traefik.docker.network=cryptlex-on-premise_backend
       - traefik.port=4200
 
+  admin-portal:
+    image: cryptlex/admin-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
```
Comment (adnan-kamili) [inline docker-compose.yml]: does this file exist now?

**Comment by saalim-mushtaq** [comment]
@adnan-kamili

File: .env [hunk 2]
```diff
@@ -10,6 +10,9 @@ EMAIL=joe.doe@mycompany.com
 WEB_API_DOMAIN=cryptlex-api.mycompany.com
 DASHBOARD_DOMAIN=cryptlex-app.mycompany.com
 RELEASE_SERVER_DOMAIN=cryptlex-releases.mycompany.com
+ADMIN_PORTAL_DOMAIN=cryptlex-admin.mycompany.com
```
Comment (adnan-kamili) [inline .env]: admin-portal.mycompany.com

and similarly for others

Comment (saalim-mushtaq) [reply .env]: @adnan-kamili  do we have to remove the dashboard.env file?

Comment (saalim-mushtaq) [reply .env]: what about  "WEB_API_DOMAIN" "DASHBOARD_DOMAIN"  and "RELEASE_SERVER_DOMAIN"

Comment (adnan-kamili) [reply .env]: these need not be changed

Comment (adnan-kamili) [reply .env]: > @adnan-kamili do we have to remove the dashboard.env file?

After customers migrate to the new portal, this file can be removed. In on-premise deployment docs we don't have to mention this file

**Review by adnan-kamili — APPROVED** [review]

**Pull request merged**